packages/ui/src/components/Header/index.tsx

```diff
@@ -118,4 +112,39 @@ const IconButtonStyled = styled(IconButton)(
 `
 )
 
-export default Header
+const LinkStyled = styled(Link)(
+  ({ theme }) => `
+  color: white;
```

Comment:
can I ask where this all comes from? The white should probably come form the theme.pallet.primary.white or something

Reply:
I styled the Link from react router similar to the Button component from mui library as we are not using the class anymore.